gator.Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM golang:1.17-alpine AS builder
+
+COPY . /tmp/gatekeeper
+
+RUN apk add --no-cache \
+    upx
+
+WORKDIR /tmp/gatekeeper/cmd/gator
+
+RUN go get -d -v
+RUN CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build -ldflags="-w -s" -o /gator
```

Comment:
can we pass in GOOS GOARCH similar to https://github.com/open-policy-agent/gatekeeper/blob/master/Dockerfile#L17-L19